src/Parameters.jl

```diff
@@ -342,7 +342,7 @@ included in `specification_dict` under key `:type`.
 Example
 =======
 
-```jldoctest
+```jldoctest; filter = [r".*Dict{Symbol.*", r".*:type       => Closure.*", r".*:c          => 3.*", r".*:subclosure => Dict{Symbol.*"]
```

Comment:
wha

Reply:
the dictionary output comes out in different order depending on the OS
(I noticed this in the tests)

so we don't need to test this....

`filter = ....` 

is regular expressions that the doctest won't include in the test but they will appear in the docstring to help the users look how the dictionary should look like!

Reply:
the perils of doctesting... i guess there's no way to automatically just print whatever the current docs build outputs